### DISCUSSION TITLE
What's the purpose of vscroll / virtual scroll in the table

### DISCUSSION THREAD

**Discussion by levyitay** [mitmproxy/mitmproxy#6488] [Q&A] [answered]
Hi all, 
I was wondering what is the purpose of scroll calculating and virtual scroll in the FlowTable?

Thank you!

**Comment by mhils** [comment]
That's a performance optimization. :)

**Reply by levyitay** [reply, marked as answer]
What exactly does it do? 
Is it to render only flows rows that are visible?

**Reply by mhils** [reply]
Correct.
